python-squid/src/main/java/org/sonar/python/tree/PyTree.java

```diff
@@ -72,4 +84,13 @@ public Tree parent() {
   protected void setParent(Tree parent) {
     this.parent = parent;
   }
+
+  abstract List<Tree> childs();
```

Comment:
I have no idea whether this fits with our naming convention, but a few ideas to avoid using the name 'childs'
getChildren()
computeChildren()
retrieveChildren()
descendants()
infants()
...kids() ? 😜 
Although I guess we can live with childs

Reply:
computeChildren it is !